system/jlib/jtrace.cpp

```diff
@@ -171,6 +171,11 @@ class CSpan : public CInterfaceOf<ISpan>
         return spanID.get();
     }
 
+    const char * getTraceID() const
+    {
+        return traceID.get();
```

Comment:
This shouldn't really be stored for non-server spans, similar to caller and local spans.  The span class structure would benefit from some refactoring (but as a separate PR)